src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionPoolGroupProviderInfo.cs

```diff
@@ -2,26 +2,27 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.Data.Common;
 using Microsoft.Data.ProviderBase;
 
 namespace Microsoft.Data.SqlClient
 {
     sealed internal class SqlConnectionPoolGroupProviderInfo : DbConnectionPoolGroupProviderInfo
     {
         private string _alias;
-        private System.Security.PermissionSet _failoverPermissionSet;
         private string _failoverPartner;
         private bool _useFailoverPartner;
+#if NETFRAMEWORK
+        private System.Security.PermissionSet _failoverPermissionSet;
+#endif
 
         internal SqlConnectionPoolGroupProviderInfo(SqlConnectionString connectionOptions)
         {
             // This is for the case where the user specified the failover partner
-            // in the connection string and we have not yet connected to get the 
+            // in the connection string and we have not yet connected to get the
             // env change.
             _failoverPartner = connectionOptions.FailoverPartner;
 
-            if (ADP.IsEmpty(_failoverPartner))
+            if (string.IsNullOrEmpty(_failoverPartner))
```

Comment:
NIT: Could you use the expression body here and for similar properties?
`internal string FailoverPartner => _failoverPartner;`